docs/en/metrics/metrics-installation.asciidoc

```diff
@@ -1,100 +1,107 @@
 [[install-metrics-monitoring]]
 [role="xpack"]
-== Install Metrics
+== Get started
 
-The easiest way to get started with Elastic Logs is by using our hosted {es} Service on Elastic Cloud.
-The {es} Service is available on both AWS and GCP, and automatically configures {es} and {kib}.
+To use the {metrics-app}, you need {es} for storing and searching your data, and {kib}
+(version 6.5 or later) for visualizing and managing it.
 
-NOTE: If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
+To ingest data, you can use {metricbeat} installed on each server you want to monitor, or
+third-party collectors that are configured to ship ESC-compliant data. <<metrics-fields-reference>>
```

Comment:
I haven't heard of ESC. Did you mean ECS? Also, a missing 'the'
```suggestion
third-party collectors that are configured to ship ECS-compliant data. The <<metrics-fields-reference>>
```

Reply:
Thanks, I did indeed mean ECS.